wordpress-mobile/WordPress-iOS
# Empty States: Replace notifications and media images.

Fixes #n/a

This replaces the images shown when:
- There is no Media.
- There are no Notifications.

To test:
- On a site with no media, go to the media library. 
- Verify the image is smaller (specifically, 112 x 86).

![media](https://user-images.githubusercontent.com/1816888/48568326-59081e00-e8bc-11e8-98da-3e87d4b79424.png)

- Go to Notifications. Select a tab that has no notifications. 
- Verify the image is smaller (specifically, 184 x 140).

![notifications](https://user-images.githubusercontent.com/1816888/48568410-8e147080-e8bc-11e8-83f5-ee04207653d0.png)
